utilities/install-collector.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# netdata
+# real-time performance and health monitoring, done right!
+# (C) 2022
+
+# This script installs community maintained collectors from netdata/community github repo
+
+NETDATA_DIR="/etc/netdata"
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run as root."
+  exit
+fi
+
+if [ ! -d $NETDATA_DIR ]; then
+    printf "\nPlease update NETDATA_DIR in %s with the Netdata user configuration directory as mentioned in https://learn.netdata.cloud/docs/agent/collectors/plugins.d#environment-variables\n" "$0"
+    exit 1
+fi
+
+printf "\nYou are about to install a collector that is NOT developed or maintained by Netdata. Would you like to proceed? [y/n]"
+read -r answer
+
+if [ "$answer" = "y" ] || [ "$answer" = "Y" ]; then
+    printf "\nInstalling collector from community repo..."
+else
+    printf "\n\nExiting...\n\n"
+    exit 1
+fi
+
+input=$1
+collector=${input#*/}
+suffix=".plugin/$collector"
+plugin=${input%"$suffix"}
+
+if [ "$plugin" == "python.d" ]; then
+    file_ext=".py"
+elif [ "$plugin" == "charts.d" ]; then
+    file_ext=".sh"
+elif [ "$plugin" == "go.d" ]; then
+    file_ext=".go"
+elif [ "$plugin" == "node.d" ]; then
+    file_ext=".js"
+fi
+
+charts="https://raw.githubusercontent.com/netdata/community/main/collectors/$1/$collector.chart$file_ext"
+config="https://raw.githubusercontent.com/netdata/community/main/collectors/$1/$collector.conf"
+executable="/usr/libexec/netdata/$plugin/"
```

Comment:
Like first comment, if we do not move this PR for `netdata/netdata`, we should either recognize where `netdata` is installed or allow users to give additional information about where `netdata` is installed.

Reply:
Agree - I think we should move it to `netdata/netdata`